src/main/java/frc/trigon/robot/subsystems/gripper/GripperCommands.java

```diff
@@ -0,0 +1,15 @@
+package frc.trigon.robot.subsystems.gripper;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.trigon.robot.RobotContainer;
+import org.trigon.commands.ExecuteEndCommand;
+
+public class GripperCommands {
+    public static Command getSetTargetVoltageCommand(GripperConstants.GripperState targetState) {
+        return new ExecuteEndCommand(
+                () -> RobotContainer.GRIPPER.setTargetVoltage(targetState),
+                RobotContainer.GRIPPER::stopMotor,
+                RobotContainer.GRIPPER
+        );
+    }
```

Comment:
this sets the target state, not voltage
add another command that sets the target voltage